identity_stardust/Cargo.toml

```diff
@@ -42,6 +44,8 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
+# Brings `AliasId` from iota-client into the public API of this crate.
+alias-id = []
```

Comment:
Turns I misunderstood your earlier comment regarding feature gating `from_alias_id`. I thought avoiding a dependency on `iota-client` in internal code would be impossible and therefore implemented it in this way (so users could opt in to getting the `AliasId` type in the public API). Given that it is not very hard to convert `AliasId` to an array of 32 bytes (as you pointed out), I think it can be fine to remove all usage of `AliasId` from this PR (except from the tests).